app/utils/views.js

```diff
@@ -205,20 +226,27 @@ module.exports = {
   AWAITING_CAPTURE_REQUEST: {
     view: 'errors/charge_confirm_state_completed',
     locals: {status: 'successful'},
-    analyticsPage: '/success_return'
+    analyticsPage: '/success_return',
+    terminalView: true
   },
 
   display: function (res, viewName, options) {
-    let action = lodash.result(this, viewName)
     options = options || {}
+    let action = lodash.result(this, viewName)
     options.viewName = viewName
     if (!action) {
       logger.error('VIEW ' + viewName + ' NOT FOUND')
       options = {viewName: 'error'}
       action = this.ERROR
     }
+    const redirectToServiceImmediatelyOnTerminalState =
```

Comment:
As discussed with @mrlumbu and @stephencdaly we will stick with this implementation, but we will change the name of the file as well as the name of the method, because we want to have central place to control the redirection.